manuscript/template.tex

```diff
@@ -282,29 +282,31 @@ \section{Goals}
+As an example of the efficiency gains that MBSE can achieve with a side-by-side comparison between MBSE and non-MBSE approaches.  Using an architecting process, of 4,858 information element transfers, noted that all of these transfers were done manually with non-MBSE approaches; however, 13\% of these transfers with automated with MBSE with the potential of up to 81\% should it be used for trade study and peer review tasks \cite{younse_cameron_bradley_2021}.  The SeaLion mission architecture team seeks to take advantage of these efficiency gains that MBSE can achieve.  Usage of MBSE for space-related projects is not without precedent either.  Projects have been taking advantage of MBSE such as the ExoMars mission, Euclid, Galileo, and nanosatellite programs \cite{mazzini}, \cite{esa}, \cite{nottage_corns_2012}.  CubeSat projects have also been using MBSE and have shown to “hold promise of reducing the burden of system engineering tasks” and can “promote uniformity and consistency across future CubeSat models” \cite{kaslow}.  These attributes are important to reduce workload amongst team members and also to facilitate new team members as they join future projects.  Especially useful for universities since students are not available long term due to events such as graduation.
+
+\subsection{Documents-As-Code Approach}
+Documents-as-code (Docs-as-code) refers to a philosophy that team members should be writing documentation with the same tools as code \cite{ibm_mbse}.  This allows for documentation to updated seamlessly without additional work with document tools (doctools).  The code tools would include version control (e.g., Git), issues trackers, code tools (e.g., Visual Studio Code), etc.  To do so would mean that the “following the same workflows as development teams, and being integrated in the product team. It enables a culture where writers and developers both feel ownership of documentation, and work together to make it as good as possible” \cite{ibm_mbse}.  The SeaLion team taking advantage of the aforementioned philosophy would realize the benefits of utilizing the same principles and practices used to manage software, using modern version control tools (e.g., Git), for the configuration management of mission and flight software architecture documentation, and captured in a model-based approach \cite{ibm_mbse}.  In addition, these models can stored and used persistently on a local file system without the use of cloud based services.  Proprietary services are not required to generate documentation, modify documentation, or modify models.  Similar approaches have been seen as Structurizr \cite{structurizr} and F Prime Prime (FPP) \cite{f_prime_prime}.  FPP is based on F Prime which is an open-source software framework developed by NASA's Jet Propulsion Laboratory \cite{f_prime}.  While docs-as-code has precedent, the methodologies noted are not easily accesible to those without a sufficient background in programming.
```

Comment:
Could expand on FPP as an example model-driven approach for producing flight software code that can be compiled directly onto flight hardware

Reply:
Resolved dda6156f73e0ae1611534d1311a34f2b1af3c6fe